ts/session/sending/MessageQueueInterface.ts

```diff
@@ -1,19 +0,0 @@
-import { OpenGroupMessage, OutgoingContentMessage } from '../messages/outgoing';
-import { RawMessage } from '../types/RawMessage';
-import { TypedEventEmitter } from '../utils';
-
-// TODO: add all group messages here, replace OutgoingContentMessage with them
-type GroupMessageType = OpenGroupMessage | OutgoingContentMessage;
-
-export interface MessageQueueInterfaceEvents {
```

Comment:
we shouldn't be removing these.. other parts of the system rely on them (even if they're not implemented fully)